src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -607,5 +608,13 @@
 					"\t\tsend \"Oops! Mending failed!\" to player")
 				.since("2.5.1");
 		}
+		if(Skript.classExists("org.bukkit.event.player.PlayerAdvancementDoneEvent")) {
+			Skript.registerEvent("Advancement done", SimpleEvent.class, PlayerAdvancementDoneEvent.class, "[on] advancement (done|finish)")
+				.description("Called when a player does an advancement")
+				.examples("on advancement done:",
+					"\tsend \"You did an advancement\" to player")
+				.requiredPlugins("1.12+")
```

Comment:
```suggestion
				.requiredPlugins("Minecraft 1.12 or newer")
```
Just for consistency